vllm/_custom_ops.py

```diff
@@ -117,13 +117,27 @@ def paged_attention_rocm(
     kv_cache_dtype: str,
     k_scale: torch.Tensor,
     v_scale: torch.Tensor,
+    is_navi: bool = False,
 ) -> None:
-    torch.ops._rocm_C.paged_attention(out, exp_sum, max_logits, tmp_out, query,
-                                      key_cache, value_cache, num_kv_heads,
-                                      scale, block_tables, seq_lens,
-                                      query_start_loc, block_size, max_seq_len,
-                                      alibi_slopes, kv_cache_dtype, k_scale,
-                                      v_scale)
+    torch.ops._rocm_C.paged_attention(out,
+                                      exp_sum,
+                                      max_logits,
+                                      tmp_out,
+                                      query,
+                                      key_cache,
+                                      value_cache,
+                                      num_kv_heads,
+                                      scale,
+                                      block_tables,
+                                      seq_lens,
+                                      query_start_loc,
+                                      block_size,
+                                      max_seq_len,
+                                      alibi_slopes,
+                                      kv_cache_dtype,
+                                      k_scale,
+                                      v_scale,
+                                      is_navi=current_platform.is_navi())
```

Comment:
Can we determine if we are running on navi hardware inside of the kernel? Looking at the archname in the kernel dispatching function seems reasonable.